src/platform/sdl/sdl-log.c

```diff
@@ -0,0 +1,87 @@
+/* Copyright (c) 2013-2015 Jeffrey Pfau
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+#include "sdl-log.h"
+#include <mgba/core/core.h>
+#include <mgba/core/log.h>
+#include <mgba/core/thread.h>
+#include <mgba/core/config.h>
+
+static bool _logToStdout;
+static FILE* _logFile;
```

Comment:
Please use the `VFile` API.

Reply:
This commit should solve that (and the brace style): https://github.com/mgba-emu/mgba/pull/2345/commits/4137ad686a57123ce1d578ad01d468971ffdd1b4